src/camps/models.py

```diff
@@ -317,6 +317,12 @@ def teardown_days(self):
 
     # convenience properties to access Camp-related stuff easily from the Camp object
 
+    @property
+    def year(self) -> int:
+        """Return the year of a camp"""
+        days = self.get_days("camp")
+        return int(days[0].lower.year)
```

Comment:
```suggestion
        return self.camp.lower.year
```

Reply:
Nice catch. I can remove the `year` property and just use `self.camp.lower.year`, I thought it was more difficult back when I started the PR. Thats the reason why I created the property.